integration_tests/src/main/python/aqe_test.py

```diff
@@ -18,9 +18,10 @@
 from asserts import assert_gpu_and_cpu_are_equal_collect
 from data_gen import *
 from marks import ignore_order, allow_non_gpu
-from spark_session import with_cpu_session
+from spark_session import with_cpu_session, is_databricks113_or_later
 
 _adaptive_conf = { "spark.sql.adaptive.enabled": "true" }
+db_113_cpu_bhj_join_allow=["BroadcastHashJoinExec", "ShuffleExchangeExec"] if is_databricks113_or_later() else []
```

Comment:
Please add a comment why these are being singled out for Databricks 11.3, ideally pointing to a followup issue that will preclude the need to fallback.